metadb/scrapers/musicbrainz.py

```diff
@@ -0,0 +1,44 @@
+import getpass as gt
+import sys
+import musicbrainzngs as mb
+
+#username = raw_input('Enter MusicBrainz Username: ')
+#password = gt.getpass('Password for {}: '.format(username))
+#mb.auth(username,password)
+
+mb.set_useragent(
+    "python-metadb-musicbrainz-scrape",
+    "0.1",
+    "https://github.com/MTG/metadb")
+
+def scrape(mbid):
+    try:
+        result = mb.get_recording_by_id(mbid, includes=["artists", "releases", "tags"])
+        
+        recording_name = result["recording"]["title"] #recording title
+        
+        artists = []
+        
+        for item in result["recording"]["artist-credit"]:
+            if isinstance(item, dict):
+                artists.append({"id": item["artist"]["id"], "name": item["artist"]["name"]})
+        
+        releases = []
+        release_groups = []
+        
+        for item in result["recording"]["release-list"]:
+            releases.append({"id": item["id"], "Title": item["title"], "Date": item["date"]})
+            
+            #Request for retrieving release info
+            rel = mb.get_release_by_id(item["id"], includes = ["artists", "release-groups"])
+            
+            #Check if release group already exists in the list in order to eliminate dublicate instances
+            if not ({"id": rel["release"]["release-group"]["id"], "Title": rel["release"]["release-group"]["title"]}) in release_groups:
```

Comment:
you could reduce code duplication by creating this dictionary first and then adding it to release_groups:
```python
release_group = {"id": rel["release"]["release-group"]["id"], "title": rel["release"]["release-group"]["title"]}
if release_group not in release_groups:
    release_groups.append(release_group)
```
You don't need the else statement here - these scripts don't need to print anything out